kafka-impl/pom.xml

```diff
@@ -87,6 +87,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+
```

Comment:
Could you explain why do you add the JUnit dependency?

Reply:
See `DelayedOperationTest`, this test used JUnit, I'll push a PR to fix it.

Reply:
Yes, before excluding, the JUnit comes from `grpc-testing` package.